cluster/pulumi/infra/src/observability.ts

```diff
@@ -136,7 +136,9 @@ export function configureObservability(dependsOn: pulumi.Resource[] = []): pulum
     {
       metadata: {
         name: 'observability',
-        labels: { 'istio-injection': 'enabled' },
+        // istio really doesn't play well with prometheus
+        // it seems to  modify the scraping calls from prometheus and change labels/include extra time series that make no sense
+        labels: { 'istio-injection': 'disabled' },
```

Comment:
this is the dumbest shit ever.
curling the proxy metrics from other pods returns exactly what we expect, curl from prometheus and it returns hundreds of series full of garbage, i have given up trying to understand it but at least I tried it an it works

Reply:
and it seems that istio and prometheus don't really play well together so that's recommended anyway